CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Use port 8081 for nexus health check; #10
```

Comment:
I'd like to see a little background explanation about the reason of the change. I, too, consider the nexus behaviour to return the health JSON along with an HTTP 500 as really bad practice. At least other developers should know about this.

An alternative could be a comment in the git commit message of the actual change but that was empty.

What do you think?

Reply:
The issue #10 says a lot about the problem. I think the changelog should only summarize what has changed and if you want to know more, you can read the issue.

Reply:
Hmm, okay, that totally counts. Thanks